### PR TITLE
MAINTAINERS: add @tricantivu as collaborator, CODEOWNERS: add @tricantivu for Spanish

### LINKED ISSUE #11989: MAINTAINERS: add @tricantivu as collaborator
Hi, @tricantivu! You seem to be enjoying contributing to the tldr-pages project.
You now have had five distinct pull requests [merged](https://github.com/tldr-pages/tldr/commits?author=tricantivu)!
That qualifies you to become a collaborator in this repository, as explained in our [community roles documentation](https://github.com/tldr-pages/tldr/blob/main/COMMUNITY-ROLES.md).

As a collaborator, you will have commit access to the repository.
That means you can merge pull requests, label and close issues, and perform various other maintenance tasks that are needed here and there.
Of course, all of this is voluntary — you're welcome to contribute to the project in whatever ways suit your liking.

If you do decide to start performing maintenance tasks, though, we only ask you to get familiar with the [maintainer's guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md).

So, what do you say? Can we add you as a collaborator?

Either way, thanks for all your work so far!

### PR DESCRIPTION
**Pull request by tricantivu**
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Closes #11989

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

Can you add yourself to the `MAINTAINERS.md` file?

**Comment by tricantivu** [comment]
> LGTM, Thanks for your contribution.
> 
> Can you add yourself to the `MAINTAINERS.md` file?

Yes

File: MAINTAINERS.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ If you are an owner of the organization, you can see an automated list [here](ht
 - **Vitor Henrique ([@vitorhcl](https://github.com/vitorhcl))**: [18 December 2023](https://github.com/tldr-pages/tldr/issues/11771) — present
 - **cyqsimon ([@cyqsimon](https://github.com/cyqsimon))**: [28 December 2023](https://github.com/tldr-pages/tldr/issues/11864) — present
 - **Jongwon Youn ([@korECM](https://github.com/korECM))**: [29 December 2023](https://github.com/tldr-pages/tldr/issues/11892) — present
+- **Alejandro Cervera ([@tricantivu](https://github.com/tricantivu))**: [3 January 2024](https://github.com/tldr-pages/tldr/issues/11989) — present
```
Comment (kbdharun) [inline MAINTAINERS.md:73]:
```suggestion
- **Alejandro Cervera ([@tricantivu](https://github.com/tricantivu))**: [4 January 2024](https://github.com/tldr-pages/tldr/issues/11989) — present
```

Just a minor suggestion and this PR is GTG. The date we refer to in the `MAINTAINERS.md` file is the date on which the invitation was sent and not when the issue was created.

Comment (tricantivu) [reply MAINTAINERS.md:73]: Understood. Sorry for the late reply.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**